api.ts

```diff
@@ -152,15 +152,15 @@ export interface SourceFileConfiguration {
     /**
      * The compiler to emulate.
      */
-    readonly intelliSenseMode: "msvc-x86" | "msvc-x64" | "msvc-arm" | "msvc-arm64" |
+    readonly intelliSenseMode?: "msvc-x86" | "msvc-x64" | "msvc-arm" | "msvc-arm64" |

```

Comment:
Actually, this requires a version update because a config provider cannot know when it is safe to omit this field any other way.  The only way it can know is by requesting a specific version of the api and getting that same version back from cpptools. If the provider gets an older version back from cpptools, then it means that cpptools has not been updated and doesn't support the field as optional.